packaging/etc/engine-config/engine-config.properties

```diff
@@ -550,6 +550,9 @@ LiveSnapshotPerformFreezeInEngine.type=Boolean
 LiveSnapshotFreezeTimeout.description=The live snapshot command without memory timeout in minutes
 LiveSnapshotFreezeTimeout.type=Integer
 LiveSnapshotFreezeTimeout.validValues=1..3000
+VolumeUtilizationChunkInMB.description="Size of extension chunk in megabytes. Use higher values to extend in bigger chunks.."
+VolumeUtilizationChunkInMB.type=Integer
+VolumeUtilizationChunkInMB.validValues=1..10000
```

Comment:
Using 1 MiB chunk does not make sense, and 10,000 MiB is too much. Use 1024..4096.

Reply:
Done.
Changed to:
`VolumeUtilizationChunkInMB.validValues=1024..4096`